グループ作成に必要なモデルとコントローラー作成

Hiro3nori3 opened the pull request:
・groupモデルの作成
・groupsコントローラーの作成
・group_userモデルの作成
・group_usersコントローラーの作成

Commit:
#WHAT　グループ作成に必要なモデルとコントローラー作成

・groupモデルの作成
・groupsコントローラーの作成
・group_userモデルの作成
・group_usersコントローラーの作成

Commit:
#WHAT groupsコントローラー内の機能名だけ

・groupsコントローラー内で必要な機能の定義を書いた

Commit:
#グループ機能に関するルーティングの記述

・グループ機能を作るにあたり必要なルーティングを記述した

Commit:
#WHAT ビューの素材のダウンロードと調整

・ビュー素材ダウンロード
・フォルダ再配置とhaml変換
・rake db:migrate(やり忘れてた)
・ルーティングにnew追加（やり忘れてた）

ta910 commented on app/models/group.rb:
```diff
@@ -0,0 +1,16 @@
+  def index
+  end
+
+	def new
```
全部インデント揃えましょう！

Hiro3nori3 replied:
なぜかmodel内にメソッド定義していたので、コントローラー内に書き換えてインデント揃えました！

ta910 commented on config/routes.rb:
```diff
@@ -3,4 +3,5 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'messages#index'
   resources :messages, only: [:index]
+  resources :groups, only: [:index, :new, :edit, :create, :update]
```
アクションの順番はindex,new,create,edit,updateとしましょう！

Hiro3nori3 replied:
おっけーです！

ta910 commented:
testディレクトリ以下は不要なので削除し、testやhelperファイルを新たに生成しないように設定しておきましょう！！

ta910 commented:
あとscssは&を使って簡単になるよう書き換えておきましょう！

Hiro3nori3 commented:
testディレクトリ以下は手動で削除して、config/application.rbに不要なファイルが生成されないような記述を付け足しました！確認お願いします！

sassも修正しました。

Commit:
#WHAT

#前コミット修正
・testディレクトリ削除
・rails gした際に不要なファイルが加わらないようにconfig/application.rbに記述
・groups_controller内に機能名を定義
・

Commit:
#WHAT Sass修正

・Sassの中身を修正

ta910 commented on config/application.rb:
```diff
@@ -14,3 +14,14 @@ class Application < Rails::Application
   end
 end
+# _test,_helper,_coffeeなどのファイルを生成しないに
+module PairsLike
```
ここ、chatspaceでは、、？

Hiro3nori3 replied:
ウェブ上のコピペしたからこうなった次第ですね！直します笑

Commit:
#WHAT 修正vol.2

・config/application.rbの中のクラス名をchatspaceに変更
